docs/guide/docs/infrastructure/pipeline.md

```diff
@@ -5,19 +5,21 @@ title: Bot Pipelines
 
 ## Overview
 
-**Bot Pipelines** (not to be confused with Development Pipelines) are built on top of the multi-bot capability of Botpress. They allow teams to work on bots with multiple **stages** the same way others would work on software products. Different versions of a bot can run and co-exist on different stages. In software development, each team has its own way of working with pipelines and reacting to events along the pipeline, Botpress pipelines let you do the same.
+**Bot Pipelines** (not to be confused with Development Pipelines) are built on top of the multi-bot capability of Botpress. They allow teams to work on chatbots with multiple **stages** just like they work on software products. Different versions of a chatbot can run and co-exist on various stages. In software development, each team has its way of working with pipelines and reacting to events along the channel. Botpress pipelines let you do the same.
```

Comment:
Totally disagree with the word channel here.
1- it's not developer jargon
2- it's super confusing with the Botpress channel concept